### ISSUE TITLE
Failed to find default file in app artifacts folder while deploying the Office Add-In UI code 

### ISSUE THREAD

**Issue by asad-shaikh-ivp** [Azure/static-web-apps#232]
Getting the following error while trying to build my code for deployment on the azure static web app. _Failed to find a default file in the app artifacts folder (dist). Valid default files: index.html,Index.html._ I am trying to deploy my OneNote Add-In code which generates a _taskpane.html_ file instead of index.html in the dist folder. Is there a way to deploy the OneNote Add-In code which do not generate an index.html file or if it's possible to update the default file in the _workflow run_ file. Any help on this would be appreciated. Thanks

**Comment by anthonychu** [comment]
Is there a way to generate an empty index.html so it passes this validation? Perhaps we should consider making it a warning instead of an error when index.html is missing. @miwebst

**Comment by miwebst** [comment]
Good question. If it is simple to add an index.html to the output that is probably the fastest mitigation. As for the long-term solution we will need to discuss this further. This check helps catch a lot of configuration issues so I'd prefer not to remove it right now. However we can certainly evaluate the possibility of including this in our configuration file.

**Comment by asad-shaikh-ivp** [comment]
The build is triggered automatically once the branch is updated and the build package I suppose is generated during the process. Is it possible to add an index.html file in the dist folder? Also if there could be a config property in the workflow file which would accept the default file, something like _default_file: taskpane.html_ would be helpful.

**Comment by miwebst** [comment]
Yes we will think about allowing other default files. In the mean time, do you have an example repo for this? I might be able to point you in the right direction for where to add an index.html file.

**Comment by asad-shaikh-ivp** [comment]
Here's the repo [link](https://github.com/asad-shaikh-ivp/OneNote-AddIn)

**Comment by smakinson** [comment]
@anthonychu @miwebst I'm looking at a similar concern. I have a TVML app and the build output is a `.js` file.

**Comment by sebastian-lenz** [comment]
I would like to perform a language redirect on the index route "/" using an Azure function and therefore don't need no real index file present there. It's odd that the system enforces such a thing.
